spec/workers/modify_project_updated_at_worker_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ModifyProjectUpdatedAtWorker do
+    let(:worker) { described_class.new }
+    let(:project) { create :project }
+
+    describe 'project touch timestamp' do
+        before do
+          allow(Project).to receive(:find).and_return(project)
+        end
+    
+        it 'touches the project timestamp on update' do
+          expect(project).to have_received(:touch).once
+          worker.perform(project)
```

Comment:
i think you'll need to setup a spy on the touch method to use `.to have_received` properly, it's different from `.to receive`. Also you'll need to adjust the order call your code then check the `have_received` expectation
```suggestion
          allow(project).to receive(:touch)
          worker.perform(project.id)
          expect(project).to have_received(:touch).once
```